### ISSUE TITLE
Convert all image embeds in content folder to use knitr::include_graphics()

### ISSUE THREAD

**Issue by bensoltoff** [cis-ds/course-site#265]
Allows for proper headers and sizing. Easy, but tedious

**Comment by djanvier** [comment]
Also wondering if you want me to abstain from any examples that uses `tidymodels` since weren't able to fully incorporate it this time around

**Comment by djanvier** [comment]
Sorry to post so close to Christmas. Feel free to reply whenever is most convenient. As I'm going through each page, I'm starting to have a problem now where the `.Rmarkdown` documents from my branch don't even line up with the pages from last quarter. Even after I pull the most recent changes, some of the publish dates go as far back as 2018. Is this a problem on my end?

**Comment by djanvier** [comment]
I just did a first past of replacing all the images with `knitr::include_graphics()` and wanted to double check something. In the two or three cases where `knitr::include_graphics()` was already used, I noticed you opted for:

```r
# static folder NOT in filepath
knitr::include_graphic(path = "/img...", error = FALSE
```
Whereas I used
```r
# static folder in filepath
knitr::include_graphics(path = "static/img..."
```

Do both work or should I change mine?

**Comment by djanvier** [comment]
Also, I noticed you set other code chunk options in that case (`fig.align='center'` and `out.width="85%"`). Do I need to do this as well?

**Comment by bensoltoff** [comment]
@djanvier I used

```r
knitr::include_graphic(path = "/img...", error = FALSE)
```

because of how `include_graphic()` generates the relative filepath. All the images are currently stored in the `static/img` folder, but Hugo (the web site generator program) automatically copies the contents of `static` into the top-level directory of the web site. So in the web pages, you must reference the `img/` folder directly.

The problem is that when you knit the page, `knitr` is still using the current working directory to build relative filepaths and based on that you **do need to include `static` in the filepath** (this is like advanced tier file paths). Hence I also omit the use of `here::here()` since in the conversion from RMarkdown to Markdown to HTML, that will build an incorrect filepath.

`include_graphics()` will check to see if the file specified in `path` exists. If it does not, it generates an error. Normally this is a good thing, but with `blogdown` we are specifically referencing a file in its "incorrect" location. So `error = FALSE` is necessary to prevent this error. [This issue documents the conundrum thoroughly](https://github.com/rstudio/blogdown/issues/184).

Long story short is I believe you need to use the my approach above, otherwise you run into problems.

**Comment by djanvier** [comment]
> @djanvier I used
> 
> ```r
> knitr::include_graphic(path = "/img...", error = FALSE)
> ```
> 
> because of how `include_graphic()` generates the relative filepath. All the images are currently stored in the `static/img` folder, but Hugo (the web site generator program) automatically copies the contents of `static` into the top-level directory of the web site. So in the web pages, you must reference the `img/` folder directly.
> 
> The problem is that when you knit the page, `knitr` is still using the current working directory to build relative filepaths and based on that you **do need to include `static` in the filepath** (this is like advanced tier file paths). Hence I also omit the use of `here::here()` since in the conversion from RMarkdown to Markdown to HTML, that will build an incorrect filepath.
> 
> `include_graphics()` will check to see if the file specified in `path` exists. If it does not, it generates an error. Normally this is a good thing, but with `blogdown` we are specifically referencing a file in its "incorrect" location. So `error = FALSE` is necessary to prevent this error. [This issue documents the conundrum thoroughly](https://github.com/rstudio/blogdown/issues/184).
> 
> Long story short is I believe you need to use the my approach above, otherwise you run into problems.

That makes a lot of sense. Thank you for clearing this up! I will making these changes asap.
